src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.spec.ts

```diff
@@ -14,11 +15,15 @@ describe(RundownHeaderContextMenuComponent.name, () => {
   })
 })
 
-async function configureTestBed(params: { mockedRundownService?: RundownService } = {}): Promise<RundownHeaderContextMenuComponent> {
+async function configureTestBed(params: { mockedRundownService?: RundownService; mockedDialogService?: DialogService } = {}): Promise<RundownHeaderContextMenuComponent> {
```

Comment:
You can also use `,` for attribute separation in type definitions.

Reply:
I know, but the linter won't let me :-D
<img width="781" alt="image" src="https://github.com/tv2/ng-sofie/assets/32238291/ac9d77cc-58ba-4493-9298-3f7876894193">